bazel/README.md

```diff
@@ -874,7 +874,7 @@ to run clang-format scripts on your workstation directly:
 
 To run the tools directly, you must install the correct version of clang. This
 may change over time, check the version of clang in the docker image. You must
-also have 'buildifier' installed from the bazel distribution.
+also have 'buildifier' installed from the bazel distribution and `aspell`.
```

Comment:
im not sure if this is the correct place to put this - given that currently aspell is required for running the python script and isnt activated via bazel - altho this may change after

Reply:
I do not find a good place to put it, maybe a new line for it?

Reply:
i think that information about aspell should not be in this file - there is no bazel job that depends on it - this file is mostly about setting up a bazel environment - info on linting is not relevant here and is wierdly misplaced

that said - its already like that and there is information about running the spell checker below

perhaps add it there - something like "if you wish to run spellchecking please install aspell"

even that tho doesnt really cover it - eg if you install on debian/ubuntu aspell as so `apt install --no-install-recommends aspell` it wont work correctly - as you need the en dictionaries that are pulled in as recommends

i think this info - like the information about linting - has no place on this page

Reply:
This section is "Running clang-format without docker", it covers more things indeed, why not rename it as something like "Running format without docker" and comment its dependence like `aspell`?

Reply:
> rename it as something like "Running format without docker" 

my preference would be "Linting your code without docker"

i still think we need to move this section - but agree its beyond the scope of this change

if you want to add information about the aspell dependency - please make it clear that its required for running the `check_spelling.py` script - not for running the other format tools

Reply:
Done.

let us do it step by step :)